sympy/sets/sets.py

```diff
@@ -924,7 +915,7 @@ def __new__(cls, start, end, left_open=False, right_open=False):
             raise ValueError("Non-real intervals are not supported")
 
         # evaluate if possible
-        if (end < start) == True:
+        if (end >= start) == False:
```

Comment:
This kind of code `if (end < start) == True` is really a workaround for not having `is_ge`. This should really be
```
if is_lt(end, start):
```